resources/tentative/charts/observable-plot.js

```diff
@@ -0,0 +1,265 @@
+import * as Plot from "@observablehq/plot";
+import { csvParse } from "d3-dsv";
+import * as d3Array from "d3-array";
+import { format as d3Format } from "d3-format";
+
+// These 2 imports use some vite machinery to directly import these files as
+// strings. Then they are constants and this reduces the load on the GC.
+import airportsString from "./datasets/airports.csv?raw";
+import flightsString from "./datasets/flights-airports.csv?raw";
+
+const ROOT = document.getElementById("chart");
+const DEFAULT_OPTIONS = {
+    // This width is really a max-width: the plot adjusts using the available
+    // width as well.
+    width: 2000,
+    // The height is especially used for the ratio.
+    height: 1000,
+};
+
+let preparedData;
+function prepare() {
+    /**
+     * AirportInformation: { state, iata, name, city, country, latitude, longitude }
+     * airports: Array<AirportInformation>
+     * flights: Array<{ origin, destination, count }>
+     */
+    const { airports, flights } = parseAirportsInformation(airportsString);
```

Comment:
This function doesn't seem to take any arguments?

Reply:
Indeed, thanks. I moved back and forth with different styles.